### ISSUE TITLE
Send API logs to opensearch

### ISSUE THREAD

**Issue by RMCampos** [bcgov/nr-spar#203]
**Describe the task**
Enable SPAR backend Services to send application logs to OpenSearch.

This issue is linked to Jira task [FSADT2-655](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-655)

**Acceptance Criteria**
- [x] Send Backend-API logs to SPAR OpenSearch Dashboard
- [ ] Send Oracle-API logs to SPAR OpenSearch Dashboard

**Additional context**
- Logs are written to a file AND to the console at the same time. The tools used for that is Logback.
- Fluentbit should read logs from `/logs` pvc on OpenShift and send them on
- SPAR OpenSearch Dashboard: https://apm.io.nrs.gov.bc.ca/_dashboards/goto/83dd3f1b6ace85a086c43221b61d71a7?security_tenant=FSA_Modernization_Program

**Comment by RMCampos** [comment]
PR #202 is related to this issue!

**Comment by RMCampos** [comment]
Blocked due to missing AWS authentication keys and secrets. Also we need to test it locally, with Funbucks, before shipping this up.

**Comment by RMCampos** [comment]
Unblocked after a meeting with OneTeam on July, 18th. Grace provided feedback on the log parsing and some issues that need to be fixed. Those are:
- Do not send messages with double quotes, only single quotes.
- Change `event.dataset` to `application.log.utc`
- Change path files, both DB and Log Files
- Remove items that are already being sent on the logs
- Add `Add service.name spar_postgres_api` to the filters
- Add `Add service.type postgres_api` to the filters

**Comment by RMCampos** [comment]
All related pain points and troubleshooting notes were added here: https://github.com/bcgov/nr-spar/wiki/Observability-with-OpenSearch-from-OpenShift#troubleshooting.

Finally this task is finished.

![Image](https://github.com/bcgov/nr-spar/assets/2219519/b4a8cd21-9c33-4a23-8d83-e1a4e60a0ac3)

**Comment by RMCampos** [comment]
Created a separate task for Oracle API: https://github.com/bcgov/nr-spar/issues/386

**Comment by RMCampos** [comment]
Some updates were made in the filter and parser files. Related PR on Funbucks: https://github.com/bcgov-nr/nr-funbucks/pull/463
